ci-operator/step-registry/redhat-developer/rhdh/helm-aks-nightly-mapt-create/redhat-developer-rhdh-helm-aks-nightly-mapt-create-commands.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
```

Comment:
```suggestion
set -e
```
so that job can terminate in from pre step in case of failure

Reply:
That would make a difference only for ingesting the values from the vault, but why not.